api/docs/http_api_index.md

```diff
@@ -1,13 +1,43 @@
 V1 API Services
 ---------------
 
+The V1 API has been removed. All previous routes can now be found under the
+[Master V3](#Master-V3-HTTP-API) API or the [CA V1](#CA-V1-HTTP-API).
+
+V2 HTTP API
+-----------
+The V2 API has been removed. All previous routes can now be found under the
+[Master V3](#V3-HTTP-API).
```

Comment:
@kylog Whoops, totally missed that the individual endpoint docs need to have changes. Those will be part of this PR - I'll push that up shortly.

Re: V1 and V2 sections - I think I agree that since we are removing those APIs it makes sense to just have a single "V1/V2 HTTP API" section. I like the idea of linking to 3.7.x docs there, but I don't know what exactly to link to - would it just be to https://docs.puppetlabs.com/references/3.7.latest/developer/file.http_api_index.html?